### ISSUE TITLE
"vue/no-unused-properties" work incorrect with useVModels from "@vueuse/core"

### ISSUE THREAD

**Issue by galsim** [vuejs/eslint-plugin-vue#2062]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** ^8.22.0
- **eslint-plugin-vue version:** ^9.3.0
- **Node version:** v16.17.1
- **Operating System:** macOS 13.0.1

**Please show your full configuration:**

```json5
/* eslint-env node */
require('@rushstack/eslint-patch/modern-module-resolution')

module.exports = {
  root: true,
  'extends': [
    'plugin:vue/vue3-essential',
    'eslint:recommended',
    '@vue/eslint-config-typescript'
  ],
  parserOptions: {
    ecmaVersion: 'latest'
  },
  rules: {
    "vue/no-unused-properties": ["error", {
      "groups": ["setup", "props"],
    }]
  }
}

```

**What did you do?**	

```vue
<script lang="ts">
import { defineComponent } from 'vue'
import { useVModels } from '@vueuse/core'

export default defineComponent({
  props: {
    msg: {
      type: String,
      required: true
    }
  },
  setup(props,{ emit }) {
    const {
      msg: msgModel
    } = useVModels(props, emit)

    return {
      msgModel
    }
  }
})
</script>
```

**What did you expect to happen?**
I expected no eslint errors, because I use msg props through useVModels

**What actually happened?**
<img width="701" alt="image" src="https://user-images.githubusercontent.com/44705564/207537704-dfa27343-10de-44ce-bc62-bdf518b61bfb.png">

**Repository to reproduce this issue**
https://github.com/galsim/no-unused-properties-use-v-models

**Comment by FloEdelmann** [comment]
I don't think we should support [this specific function](https://vueuse.org/core/useVModels/) from VueUse. But we should probably consider all props used when they are passed to a function that is not declared in the same file.

**Comment by galsim** [comment]
Any updates here?

**Comment by FloEdelmann** [comment]
No, but PRs are welcome :wink:
